lib/factory_bot/enum.rb

```diff
@@ -12,14 +23,16 @@ def build_traits(klass)
       end
     end
 
+    attr_reader :attribute_name
```

Comment:
Are we using this anywhere?

Reply:
This was being used in `lib/factory_bot/definition.rb` to help with not redefining traits in the automatically_register_defined_enums method, it is not used in the `Definition#register_enum` method as the key in the `registered_enum` hash

Reply:
this is now also being used in the registered_enums hash as the key to look up to ensure enums are not registered twice